src/language.cpp

```diff
@@ -0,0 +1,33 @@
+#include "language.h"
+#include <QFile>
+#include <QJsonDocument>
+#include <QJsonObject>
+#include <QDebug>
+
+Language::Language()
+{
+}
+
+void Language::setTranslationsFromJsonFile(const QString &filePath)
+{
+    QFile file(filePath);
+    if (!file.open(QIODevice::ReadOnly)) {
+        return;
+    }
+    QJsonDocument doc = QJsonDocument::fromJson(file.readAll());
+    file.close();
+    if(doc.isNull()) {
+        qInfo() << "Invalid JSON...\n";
+        return;
+    }
```

Comment:
We should do something here. No translation means no text at all to display.
(If getText handle missing key, we can do nothing here)